app/main/views/notifications.py

```diff
@@ -48,23 +47,9 @@ def user_research_consent():
         user = data_api_client.get_user(current_user.id)
         form = UserResearchOptInForm(user_research_opt_in=user['users']['userResearchOptedIn'])
 
-    # Set the seen_user_research_message cookie if it does not exist.
```

Comment:
not sure of the history here. @katstevens can you confirm this is safe to delete?

Reply:
Oh yeah we don't need this we got rid of this in https://github.com/alphagov/digitalmarketplace-user-frontend/pull/204

Reply:
Ditch it! 🔥